web/src/components/AppBar/AccountTab.js

```diff
@@ -186,7 +188,7 @@ const AccountList = ({
 	const onHandleRoutes = (value = '/', title = '') => {
 		const selectedTab = {
 			'LOGIN.HELP': () => setIsHelpResources(true),
-			'ACCOUNTS.TAB_SIGNOUT': () => removeToken(),
+			'ACCOUNTS.TAB_SIGNOUT': () => setIsLogout(true),
```

Comment:
whats this change about? @jsmitrah

Reply:
Operators cannot close the confirmation form after receiving notification,
The issues were mentioned in the design channel